test/unit/model/include.test.js

```diff
@@ -179,7 +180,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           include: [{ model: this.Project }],
         });
 
-        expect(options.include[0]).to.have.property('where').which.deep.equals({ active: true });
+        expect(options.include[0]).to.have.property('where').which.deep.equal({ active: true  });
```

Comment:
You reverted your usage of deepEqual, is that intended? Didn't you add it to be able to compare `symbol`?

I saw a commit fail because it was trying to compare 'object with null prototype' to 'object', I assume that's why you reverted?

https://github.com/chaijs/chai/issues/1054 has had some movement but I don't think they'll release the change until chai 5

---

also if `deep.equals` and `deep.equal` are the same, can you revert to `deep.equals` to reduce risk of conflict?